tests/run-pass/change_current_dir.rs

```diff
@@ -0,0 +1,14 @@
+// ignore-windows: TODO the windows hook is not done yet
+// compile-flags: -Zmiri-disable-isolation
```

Comment:
Okay now there's enough stuff here to warrant its own file. :)
I'd prefer `current_dir.rs` as filename though, as it tests more than just changing it.

Reply:
https://github.com/rust-lang/miri/pull/983/commits/8368d4f2b44ae1441390a2f7eee48af340c8d70c